tests/Frontend/PathSpec.hs

```diff
@@ -33,6 +35,19 @@ import Network.Wai
 import Test.Hspec.Wai (get, post, shouldRespondWith)
 import qualified Test.Hspec.Wai.QuickCheck as Wai (property)
 
+isBrokenPath :: Main -> Bool
```

Comment:
> > +isBrokenPath :: Main -> Bool
>
> As far as I know, there is no DB read involved in the `Paths and handlers` test suite, it uses the `mockAulaMain` uses only arbitrary instances to generate some mock data for the handler. I am opening an issue about it.

i really like this test, almost as much as i dislike our generic Arbitrary instances.  i think the problem in this case was that arbitrary generates urls to comments of arbitrary nesting depth (not maximum 2), but similar problems keep kreeping up.

you are right, though: we're not using the database state here, and i'm not sure how we would take an AulaData value generated based on stories and generated arbitrary valid paths from that.  nor that that would be the same test.  i'll give this another go.

testing is exhausting!  :/